css/selectors/cue.json

```diff
@@ -0,0 +1,59 @@
+{
+  "version": "1.0.0",
+  "data": {
+    "css": {
+      "selectors": {
+        "::cue": {
```

Comment:
`"cue"` not `"::cue"`
(our export should be `css.selector.cue` not `css.selectors.::cue`)